src/dashboard/Push/PushDetails.react.js

```diff
@@ -39,7 +39,13 @@ const EXP_STATS_URL = 'http://docs.parseplatform.org/ios/guide/#push-experiments
 let getMessage = (payload) => {
   if(payload) {
     let payloadJSON = JSON.parse(payload);
-    return payloadJSON.alert ? payloadJSON.alert : payload;
+		if (payloadJSON.alert.body) {
```

Comment:
So under PushDetails if an alert object contains title and body, would you like me to show both the title and message just below the title in smaller font size?